src/iconc/cmain.c

```diff
@@ -124,7 +124,16 @@ char **argv;
       refpath = patchpath+18;
 #endif					/* ExpTools */
    else
-     refpath = relfile(argv[0], "/../");
+      refpath = relfile(argv[0], "/../");
+
+     if (refpath[strlen(refpath)] != '/') {
```

Comment:
Oh, is this commented out and going away?  That is not at all clear from the view I am presented with.

Reply:
I added it in an earlier commit, but commented it out in later commit. If you are looking at the specific commit where I added it then you are correct. I do  see the label "Outdated"  in your comment above the code when you look at the comment in the PR.

Reply:
This is all a result of a work-in-progress changes, my apologies for the  confusion.